Jenkinsfile_parameterized

```diff
@@ -6,7 +6,7 @@ properties([
     string(name: 'PRODUCT_NAME', defaultValue: 'bar', description: ''),
     string(name: 'APP', defaultValue: 'web', description: ''),
     string(name: 'TYPE', defaultValue: 'nodejs', description: ''),
-    string(name: 'ENVIRONMENT', defaultValue: 'sandbox', description: 'Environment where code should be build and deployed'),
+    string(name: 'ENVIRONMENT', defaultValue: 'saat', description: 'Environment where code should be build and deployed'),
```

Comment:
you put a choice to hold saat/sprod/sandbox if you want .. otherwise leave as free text as it is

Reply:
I didn't get you. All I need is saat env to be built by default. If left as free text which environment will be built automatically?

Reply:
it's already free text.. i'm suggesting to put a choice box instead to hold three entries (saat/sprod/sandbox) in which case "saat" is the default one.